layout/js/animate.js

```diff
@@ -130,7 +130,7 @@ function openNWIS(id, event){
  if( /Android|webOS|iPhone|iPad|iPod|BlackBerry|IEMobile|Opera Mini/i.test(navigator.userAgent) ) {
    event.stopPropagation();
 }else{
-  window.open('http://waterdata.usgs.gov/nwis/uv?site_no='+id,'_blank');
+   window.parent.vizlab.clicklink('http://waterdata.usgs.gov/nwis/uv?site_no='+id,'_blank');
```

Comment:
Do you need the `.parent`?  I think that was just there because we were embedding svg in object tag.  You should be able to just do `vizlab.clicklink()`